tools/install-powershell.ps1

```diff
@@ -79,6 +79,65 @@ Function Remove-Destination([string] $Destination) {
     }
 }
 
+Function Test-PathInRegistryPath([string] $Path) {
+    #Remove ending DirectorySeparatorChar for comparison purposes
+    $Path = [System.Environment]::ExpandEnvironmentVariables([regex]::Replace($Path, '\\;$|\\$', ''));
+    $InstalledPaths = @()
+    #[System.Environment]::GetEnvironmentVariable automatically expands all variables
+    $InstalledPaths += @([System.Environment]::GetEnvironmentVariable('PATH', [System.EnvironmentVariableTarget]::User).Split(';'))
+    $InstalledPaths += @([System.Environment]::GetEnvironmentVariable('PATH', [System.EnvironmentVariableTarget]::Machine).Split(';'))
+    #Remove ending DirectorySeparatorChar in all items of array for comparison purposes
+    $InstalledPaths = $InstalledPaths.ForEach( { $_.Replace('\\$', '') } )
+    if ($InstalledPaths -icontains $Path) {
+        return $True
+    }
+    return $False
+}
+
+Function Add-Path([Microsoft.Win32.RegistryKey] $Key, [string] $Path) {
```

Comment:
Add a description of what each function does and the expected values of the parameters.